README.md

```diff
@@ -80,25 +80,40 @@ Current prototype supports:
 | App Service                         | app                         | azurerm_app_service                     |
 | Application Insights                | appi                        | azurerm_application_insights            |
 | App Service Environment             | ase                         | azurerm_app_service_environment         |
+| Application Security Group          | asg                         | azurerm_app_security_group              |
 | Azure Kubernetes Service            | aks                         | azurerm_kubernetes_cluster              |
 | Azure Kubernetes Service DNS prefix | aksdns                      | aks_dns_prefix                          |
 | AKS Node Pool Linux                 | aksnpl                      | aks_node_pool_linux                     |
 | AKS Node Pool Windows               | aksnpw                      | aks_node_pool_windows                   |
 | Azure Site Recovery                 | asr                         | azurerm_recovery_services_vault         |
+| Azure Availability Set              | avail                       | azurerm_availability_set                |
+| Azure Vpn Connection                | cn                          | azurerm_vpn_connection                  |
 | Azure Event Hubs                    | evh                         | azurerm_eventhub_namespace              |
 | generic                             | gen                         | generic                                 |
 | Azure Key Vault                     | kv                          | azurerm_key_vault                       |
 | Azure Monitor Log Analytics         | la                          | azurerm_log_analytics_workspace         |
+| Azure Load Balancer (External)      | lbe                         | azurerm_load_balancer_external          |
```

Comment:
ive often seen ilb vs. elb for internal/external. I know its not lb* prefix, but its fairly common.